### ISSUE TITLE
`svelte/no-unused-svelte-ignore` wrongly reports `svelte-ignore unused-export-let` as not warned (with `lang="ts"`)

### ISSUE THREAD

**Issue by divdavem** [ota-meshi/eslint-plugin-svelte#355]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.33.0

### What version of `eslint-plugin-svelte` are you using?

2.15.0

### What did you do?

```svelte
<script lang="ts">
  // svelte-ignore unused-export-let
  export let something;
</script>
```

### What did you expect to happen?

I expected no eslint error because the `something` variable is actually unused.

By the way, if I remove the `svelte-ignore` comment, I correctly get the following eslint error, proving that the `svelte-ignore` comment is actually useful:
> Component has unused export property 'something'. If it is for external reference only, please consider using `export const something`(unused-export-let) ESLint [svelte/valid-compile](https://ota-meshi.github.io/eslint-plugin-svelte/rules/valid-compile/)

Note that the problem does not happen if I remove `lang="ts"`.

### What actually happened?

I got the following eslint error:
svelte-ignore comment is used, but not warned [svelte/no-unused-svelte-ignore](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-unused-svelte-ignore/)

### Link to **GitHub Repo** with Minimal Reproducible Example

https://ota-meshi.github.io/eslint-plugin-svelte/playground/#eJwlzE0KgCAQQOGrDLMX9/2dxE3oYIKN4YwRRHfPaPnB493oSyAccBJf06GQV46zQxWHi2MAa0FOykomRS6VoHETCoauo1Q1mfSrfkEXSNlJt8RxdDzZ/9pP+LzPoCYU
